src/configs/index.ts

```diff
@@ -0,0 +1 @@
+export { AutoblocksConfig } from './config';
```

Comment:
do we want this to be AutoblocksConfigManager to match prompts or nah ?

Reply:
I was debating, but decided not to since it works differently